onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -3571,7 +3571,7 @@ def aten_fmin(self: TensorType, other: TensorType) -> TensorType:
     raise NotImplementedError()
 
 
-@torch_op("aten::fmod")
+@torch_op(("aten::fmod", "aten::fmod.Tensor", "aten::fmod.Scalar"))
```

Comment:
If an op does not have the `default` overload defined in https://github.com/pytorch/pytorch/blob/main/aten/src/ATen/native/native_functions.yaml#L9722. We can remove the default version

```suggestion
@torch_op(("aten::fmod.Tensor", "aten::fmod.Scalar"))
```